tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -36,57 +36,57 @@
 
     <string name="tvDNSStop">DNSCrypt Stopped</string>
     <string name="tvDNSInstalled">DNSCrypt Installed</string>
-    <string name="tvDNSInstalling">DNSCrypt Installing</string>
-    <string name="tvDNSNotInstalled">DNSCrypt Not Installed</string>
+    <string name="tvDNSInstalling">DNSCrypt Installing…</string>
+    <string name="tvDNSNotInstalled">DNSCrypt is Not Installed</string>
 
-    <string name="tvDNSStarting">DNSCrypt Starting</string>
-    <string name="tvDNSRunning">DNSCrypt Running</string>
-    <string name="tvDNSStopping">DNSCrypt Stopping</string>
+    <string name="tvDNSStarting">DNSCrypt Starting…</string>
+    <string name="tvDNSRunning">DNSCrypt Running…</string>
+    <string name="tvDNSStopping">DNSCrypt Stopping…</string>
     <string name="tvDNSDefaultLog" translatable="false">DNSCrypt-proxy version</string>
 
-
+    
     <string name="tvTorStop">Tor Stopped</string>
     <string name="tvTorInstalled">Tor Installed</string>
-    <string name="tvTorInstalling">Tor Installing</string>
-    <string name="tvTorNotInstalled">Tor Not Installed</string>
+    <string name="tvTorInstalling">Tor Installing…</string>
+    <string name="tvTorNotInstalled">Tor is Not Installed</string>
 
-    <string name="tvTorStarting">Tor Starting</string>
-    <string name="tvTorConnecting">Connecting</string>
-    <string name="tvTorRunning">Tor Running</string>
-    <string name="tvTorStopping">Tor Stopping</string>
+    <string name="tvTorStarting">Tor Starting…</string>
+    <string name="tvTorConnecting">Connecting…</string>
+    <string name="tvTorRunning">Tor Running…</string>
+    <string name="tvTorStopping">Tor Stopping…</string>
     <string name="tvTorDefaultLog" translatable="false">Tor version</string>
 
     <string name="ok" translatable="false">OK</string>
     <string name="cancel">Cancel</string>
     <string name="error">Error!</string>
     <string name="warning">Warning!</string>
-    <string name="no_root" translatable="false">NO ROOT!</string>
+    <string name="no_root" translatable="false">No Root!</string>
     <string name="install">Install</string>
     <string name="exit">Exit</string>
     <string name="done">Done</string>
     <string name="wrong">Something went wrong!</string>
-    <string name="install_message">Do you want to complete installation?</string>
+    <string name="install_message">Complete the installation?</string>
     <string name="premium">Premium version</string>
```

Comment:
```suggestion
    <string name="premium">Pro version</string>
```
I like "premium" better, but are those the same thing? "Pro" is used more often.